src/types/scanText.ts

```diff
@@ -5,6 +5,7 @@ export namespace ScanText {
     detectionRuleUUIDs?: string[]
     detectionRules?: Detector.Rule[]
     contextBytes?: number
+    defaultRedactionConfig?: Detector.RedactionConfig
```

Comment:
@evanfuller you can inspect the type here: https://github.com/nightfallai/nightfall-nodejs-sdk/blob/main/src/types/detectors.ts#L42. Let me know if we need to change anything.